weeks/week07/week07.md

```diff
@@ -1,3 +1,38 @@
 # Week 7: Web Automation with Playwright
 
-Coming soon.
+## General information:
+Playwright 
+It is the powerful tool for Web Test automation (UI and API) in different programming languages:
+-TypeScript, 
+- JavaScript,
+- .NET,
+- Java.
+- Python
+
+It is 
+- Cross-browser,
+- Cross-platform,
+- Support mobile testing
+
+The scope of this page: Playwright - Python
+
+## Playwright - Python General information:
+Convenient for Web and API testing
+Intuitive integration with Pytest Test Automation Framework
+Intuitive integration with Allure
+Inherited  support for all Python functionality
+Easy to use within CI/CD (e.g. Jenkins)
+
+## References:
+
+1. [Playwright - Python oficial page](https://playwright.dev/python/)
+2. [TBD](TBC/)
+3. [TBD](TBC/)
+
+---
+
+Regular expressions are powerful tools for matching, searching, and manipulating text. They are widely used in data validation, parsing, and text processing. Now suppose you need to write Python code to find out whether a log record contains the substring 'HttpStatusCode: xxx' where http code can be any three consecutive decimal digit characters.
```

Comment:
Wrong content here.

Reply:
@ericrommel - thanks for the comment. Removed.